std/json.d

```diff
@@ -96,11 +96,36 @@ enum JSON_TYPE : byte
     FALSE    /// ditto
 }
 
+private struct DefaultPolicy
+{
+    enum sortObjectMembers = true;
+}
```

Comment:
you can simplify a lot the rest of the code by always requiring a template ObjectType to Policy:
```
private struct DefaultPolicy
 { 
   enum sortObjectMembers = true;
   alias ObjectType(Value)=Value[string];
 }
```

Reply:
Which code can be simplified?